tests/unit/drivers/test_chunk2doc_rank_drivers.py

```diff
@@ -41,62 +43,46 @@ def create_document_to_score():
             match.score.value = match.id
     return doc
 
+@pytest.fixture(
+    scope="module",
+    params=[
+        MockLengthRanker(),
+        MaxRanker(),
+        MinRanker()
+])
+def ranker(request):
+    return request.param
+
+@pytest.fixture(scope="module")
```

Comment:
what other options of scope are there?

Reply:
@JoanFM Possible values for scope are: function, class, module, package or session.